autoload/sy.vim

```diff
@@ -24,9 +31,10 @@ function! sy#start() abort
     return
   endif
 
-
   function! s:chdir()
```

Comment:
Hmm, I agree. Actually it's easy to see that the cache quickly becomes stale when a few `:cd`/`:lcd`/`:tcd` commands were used since registering that buffer.

Even worse is caching the `cwd` one line below, right?